cmd/osv-scanner/main_test.go

```diff
@@ -91,6 +91,8 @@ type cliTestCase struct {
 func testCli(t *testing.T, tc cliTestCase) {
 	t.Helper()
 
+	t.Logf("Running test: [%s]", tc.name)
```

Comment:
I think this should be removed as Go already prints the name of the test if it has one, otherwise it prints its index so having this just adds extra noise:

```
--- FAIL: TestRun_LocalDatabases (0.00s)
    --- FAIL: TestRun_LocalDatabases/blah (1.01s)
        main_test.go:843: Running test: [blah]
        main_test.go:843: stdout
             got:
            Scanning dir ./fixtures/locks-many/composer.lock
            Scanned /home/jones/workspace/projects-oss/osv-scanner/cmd/osv-scanner/fixtures/locks-many/composer.lock file and found 1 package
            Loaded Packagist local db from /tmp/osv-scanner-test-4097718912/osv-scanner/Packagist/all.zip
            No vulnerabilities found

             want:
            Scanning dir ./fixtures/locks-many/composer.lock
            Scanned %%/fixtures/locks-many/composer.lock file and found 1 package
             Loaded Packagist local db from %%/osv-scanner/Packagist/all.zip
            No vulnerabilities found
FAIL
```

vs

```
--- FAIL: TestRun_LocalDatabases (0.00s)
    --- FAIL: TestRun_LocalDatabases/#11 (1.50s)
        main_test.go:843: Running test: []
        main_test.go:841: stdout
             got:
            Scanning dir ./fixtures/locks-many/composer.lock
            Scanned /home/jones/workspace/projects-oss/osv-scanner/cmd/osv-scanner/fixtures/locks-many/composer.lock file and found 1 package
            Loaded Packagist local db from /tmp/osv-scanner-test-1140406433/osv-scanner/Packagist/all.zip
            No vulnerabilities found

             want:
            Scanning dir ./fixtures/locks-many/composer.lock
            Scanned %%/fixtures/locks-many/composer.lock file and found 1 package
             Loaded Packagist local db from %%/osv-scanner/Packagist/all.zip
            No vulnerabilities found
FAIL
```

Reply:
Huh, I must have missed that when debugging, removed.